NAMESPACE

```diff
@@ -1,5 +1,6 @@
 # Generated by roxygen2: do not edit by hand
 
+export(HubbellSE)
```

Comment:
If the logic of this function is the same than in `simulateGLV` i.e. it generates time series from a given model, then we should use a consitent naming scheme. Would this be better named as `simulateHubbell`? Also add an alias `simulateNeutral`

Reply:
What does simulateNeutral mean? I get warning from that says it is unknown tag

Reply:
You can give an alias for a function by adding
`
#' @aliases simulateNeutral`

This means that you can call the same function with two different names. Here simulateNeutral may be easier to remember for some people but simulateHubbell is more exact.